02-fundamentals.Rmd

```diff
@@ -614,6 +614,33 @@ Joins and Data Blending work when it is required to append columns from one tabl
+
+!()[https://onlinehelp.tableau.com/current/guides/get-started-tutorial/en-us/Img/Drag28.png]
+
+**Element**          | **Description**
+---------------------|-------------------------------------------------------------------------------------------------------------------
+**Data Pane**        | Displays the data set that is in the view and the fields (columns) in the data set that are automatically sorted into dimentions or measures (explained below).
+**Cards**            | Cards are containers for different controlls available in Tableau. 
+Marks                | This is a main way of manipulating visual aspects of graphs and charts. You can encode your data fields with elements like type, color, size, shape etc. You can have multiple *measures* and *dimentions* added to the marks cards to add different properties such as different regions denoted by shapes and quantity denoted by size. 
+Summary              | The summary card is available on the *show/hide cards* toolbar menu to add summary statistics either for the data selection, or the entire data source. 
+**Shelves**          | Shelves are a type of card. 
+Rows and Columns     | Rows and columns are the shelves where you drop data fields in order to create the structure of your visualization. When creating a table visualization the columns will create the columns of the table, while the rows would create the rows of the table. Any number of fields can be dragged onto the shelves to create a more granular picture of the data. When a *dimention* is placed on the shelf, headers for the member of the dimention are created. When a *measure* is placed on a shelf, the quantitative axes for the measure are created. 
+Filters              | The filters shelf is where you specify the data to include and exclude by dragging *measures*, and/or *dimentions*. For example, you can put your date into the filter if you only want to see revenue from the last quarter, or you can put a categorical dimention into the filter if you only want to see data for specific products. 
+Pages                | The pages shelf allows for a series of separate page views with a common axis to better analyze how a certain *measure* affects the rest of the data. When a field is added to the pages shelf a control panel is added to the right side of the view to move through the pages (either manually or automatically).
+Measure Values       | At the bottom of the *measures* data pane is a special field that contains all measures of the data collected into one field. 
+
+
+**Calculation Type** | **Description**
+---------------------|-------------------------------------------------------------------------------------------------------------------
+**Dimention**        | Qualitative or categorical data that would normally appear as column headers for rows of data that normally defines the granularity that shows in the view, i.e. customer name or order date. 
```

Comment:
Hi Laura, 
Great addition.
I think the spelling of Dimension is incorrect. Please check. 
Thanks.

Reply:
Thanks for catching that @pragyansharma24, spelling errors have now been changed.